data/yam/agama/auto/alp_add_user.sh

```diff
@@ -0,0 +1,8 @@
+set -ex
```

Comment:
I noticed naming under `data/yam/agama/auto/` is wrong.
`alp_tw.json` -> `default_tumbleweed.json`
`alp_tw.sh` -> `default_tumbleweed.sh`
`alp.sh` -> `alp_bedrock.sh`
Could you apply that change?

For this one should be `/data/yam/agama/auto/tumbleweed_add_user.sh`
In other words, alp shouldn't be mention at all because it is not connected in any way.